packages/as-types/assembly/safeMath.ts

```diff
@@ -1,8 +1,18 @@
+/**
+ * The SafeMath class provides utility functions for basic arithmetic operations.
+ * These functions perform overflow and underflow checks to prevent unwanted behavior
+ * when dealing with unsigned 64-bit integers (u64). The SafeMath class should be used
+ * when working with arithmetic operations that require increased safety and precision.
+ */
 export class SafeMath {
   /**
+   * add - Returns the addition of two unsigned 64-bit integers (u64),
```

Comment:
'function_name - title' Is this some kind of standard way to make function titles in TSDoc ?
@Ben-Rey @gregLibert what do you think ? Should we apply this rule every where in documentation ?

Reply:
No need to repeat the name of the function in the title.
The title is just a description of what the function does
https://tsdoc.org/